src/integration/common/renovate.json

```
@@ -0,0 +1,8 @@
+{
+  "extends": ["config:base", ":pinAllExceptPeerDependencies"],
+  "packageRules": [
+    {
+      "packagePatterns": ["^@psammead/"]
+    }
+  ]
```

I think this would work for making dep PRs at the weekend but psammead dep updates daily

```suggestion
    "schedule": [
      "every weekend"
  ],
  "packageRules": [
    {
      "packagePatterns": ["^@psammead/"],
      "extends": ["schedule:monthly"]
    }
  ]
```